client/dist/index.html

```diff
@@ -5,7 +5,7 @@
   <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
-<div id="test"></div>
+<div id="feature"></div>
 <script type="text/javascript" src="./bundle.js"></script>
```

Comment:
You do not need to define the script type - in fact, it's recommended that you just use `<script src="bundle.js></script>`

Reply:
Ah gotcha, I will fix that right away